doc/quickstart.md

```diff
@@ -5,59 +5,66 @@ USB device.
 
 The Tillitis Key 1 kit includes:
 
-- Tillitis Key 1 USB device, marked MTA1-USB V1
-- Programmer board based on Raspberry Pi Pico, with a white device
-  holder/jig
+- Tillitis Key 1 USB stick with USB-C plug, marked MTA1-USB V1
+- Programmer board based on Raspberry Pi Pico, with a white holder/jig
 - USB-cable with micro-B plug, for connecting the programmer to
   computer
-- USB-C cable
+- USB-C extension cable
 - USB-C to USB-A adapter
 
 # Programming FPGA bitstream and firmware onto Tillitis Key 1
 
-Connect the programmer to the computer using the mentioned cable. It
-is convenient to connect the USB device to the USB-C cable, and then
-connect the cable to the computer. The latter using the USB-C-to-A, if
-needed.
+Connect the programmer to the computer using the USB-cable with
+micro-B plug.
 
-`lsusb` should list two new devies: `cafe:4004 Blinkinlabs ICE40 programmer`
-and `1207:8887 Tillitis MTA1-USB-V1`.
+Place the Tillitis Key 1 (the USB stick) correctly in the programming
+jig and close the hatch.
 
-The USB device is then placed correctly in the programming jig, and
-the hatch closed. The USB device can remain in the jig during repeated
-programming and testing cycles. The jig has a cutout to allow for
-touching next to the LED where the touch sensor is located.
+The USB stick can remain in the jig during repeated development,
+programming and testing cycles. The USB stick is then also connected
+to the computer using the provided USB-C cable (use the USB-C-to-A
+adapter if needed). Note that connecting the USB stick to the computer
+is not required for programming it). For this purpose, the jig also
+has a cutout to allow touching where the touch sensor is located, next
+to the LED.
+
+On Linux, `lsusb` should list the connected programmer as `cafe:4004
+Blinkinlabs ICE40 programmer`. If the USB stick is also connected it
+shows up as `1207:8887 Tillitis MTA1-USB-V1`.
 
 To install the software needed for building and programming, please
 refer to [toolchain_setup.md](toolchain_setup.md).
 
-You are now ready to generate the FPGA bitstream including the standard
-firmware, and program the flash on the connected USB device. This should be run
-as your regular non-root user, but the the programming is done (in the
-Makefile) with `sudo tillitis-iceprog` (so sudo is expected be set up).
+You are now ready to generate the FPGA bitstream (including building
+the standard firmware) and program the flash on the USB stick. The
+following should be run as your regular non-root user, but the
+programming is currently done using `sudo` (so sudo is expected to be
+set up for your user; the Makefile runs `sudo tillitis-iceprog …`).
```

Comment:
Would it not be better to recommend the user to set up proper udev rules? "Run this random program you just compiled as root" does not feel optimal IMHO. It's so easy to make the programmer usable as a regular user.

Reply:
later..